api/src/paths/project/{projectId}/survey/{surveyId}/observations/create.ts

```diff
@@ -0,0 +1,143 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../../../../constants/roles';
+import { getDBConnection } from '../../../../../../database/db';
+import { HTTP400 } from '../../../../../../errors/CustomError';
+import { PostBlockObservationObject } from '../../../../../../models/block-observation-create';
+import { observationResponseObject } from '../../../../../../openapi/schemas/observation';
+import { postBlockObservationSQL } from '../../../../../../queries/observation/observation-create-queries';
+import { getLogger } from '../../../../../../utils/logger';
+import { logRequest } from '../../../../../../utils/path-utils';
+
+const defaultLog = getLogger('paths/project/{projectId}/survey/{surveyId}/observations/create');
+
+export const POST: Operation = [
+  logRequest('paths/project/{projectId}/survey/{surveyId}/observations/create', 'POST'),
+  createObservation()
+];
+
+POST.apiDoc = {
+  description: 'Create a new observation.',
+  tags: ['observation'],
+  security: [
+    {
+      Bearer: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.PROJECT_ADMIN]
+    }
+  ],
+  requestBody: {
+    description: 'Observation post request object.',
+    content: {
+      'application/json': {
+        schema: {
+          title: 'Observation request object',
+          type: 'object',
+          required: ['observation_type'],
+          properties: {
+            observation_type: {
+              title: 'type of observation',
+              type: 'string'
+            },
+            observation_post_data: {
+              title: 'Generic observation json data',
+              type: 'object',
+              properties: {}
+            }
+          }
+        }
+      }
+    }
+  },
+  responses: {
+    200: {
+      description: 'Observation response object.',
+      content: {
+        'application/json': {
+          schema: {
+            ...(observationResponseObject as object)
```

Comment:
you need to create something new here, cannot modify the one I created for `getForUpdate` response